### PR TITLE
Feature/ab#58 app entities can be saved per app

### PR DESCRIPTION
**Pull request by DerCed**
## Description
ab#58
Entities will be saved when the Entity dialog is closed

## Motivation and Context
As a user, I will spend a lot of time prototyping the entities. Therefore it is a nice feature to save periodically the progress the use made 

## How has this been tested?
Local
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by witchpou** [comment]
At the moment, it is not possible to add fields or relations to an existing entity without any field or relation. Is it fixed? If not, please fix it or add a bug-ticket.

**Review by witchpou — CHANGES_REQUESTED** [review]

**Comment by DerCed** [comment]
> At the moment, it is not possible to add fields or relations to an existing entity without any field or relation. Is it fixed? If not, please fix it or add a bug-ticket. (see https://pb.starwit.de/ljprojectbuilder/#/app/2/edit)

This issue should be fixed now. The problem was, that during saving the list was converted to an null object

File: webclient/app/src/app/pages/appEditor/sections/erSection/ErSection.jsx [hunk 1]
```diff
@@ -181,62 +172,58 @@ function ErDesigner(props) {
...
+                open={drawerOpen}
+                onClose={closeDrawer}
+                className={erDesignerStyles.drawer}
+            >
+                <SyntaxHighlighter
+                    language="json"
+                    style={docco}
+                    showLineNumbers
+                    customStyle={{
+                        lineHeight: "1.5", fontSize: ".75em"
```
[the remark this answers is not in the document]
Comment (DerCed) [reply webclient/app/src/app/pages/appEditor/sections/erSection/ErSection.jsx:194]: Jup, but this SyntaxHighlighter was implemented way before the one in ApptemplateCard ;) 
See lines deleted lines 199 and below

[the remark this answers is not in the document]
Comment (witchpou) [reply webclient/app/src/app/pages/appEditor/sections/erSection/ErSection.jsx:194]: Please separate ticket under https://dev.azure.com/starwit/lj-projectbuilder/_workitems/edit/39 or improvement of the code ;-). Custom styles are also inside.

[the remark this answers is not in the document]
Comment (DerCed) [reply webclient/app/src/app/pages/appEditor/sections/erSection/ErSection.jsx:194]: See See https://dev.azure.com/starwit/lj-projectbuilder/_workitems/edit/39/?workitem=96

**Review by witchpou — APPROVED** [review]

**Pull request merged**